recipes/cube_helper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "cube_helper" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MetOffice/cube_helper/archive/v{{ version }}.tar.gz
+  sha256: 7f84d39f2b41165568ab643691c02c42effba8ee6cb41aaf210df83fadbed77c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - iris
```

Comment:
I don't think you need `iris` to build the package so I think it can be removed here.

Reply:
@synapticarbors thanks for the comprehensive review. 2cb387f removes Iris from the build and the tests are passing.